draft-js-mention-plugin/src/mentionSuggestionsStrategy.js

```diff
@@ -31,7 +31,7 @@ const findWithRegex = (regex, contentBlock, callback) => {
 export default (trigger: string, supportWhiteSpace: boolean, regExp: string) => { //eslint-disable-line
   const MENTION_REGEX = supportWhiteSpace ?
     new RegExp(`${escapeRegExp(trigger)}(${regExp}|\\s){0,}`, 'g') :
-    new RegExp(`(\\s|^)${escapeRegExp(trigger)}${regExp}`, 'g');
+    new RegExp(`(.|^)${escapeRegExp(trigger)}${regExp}`, 'g');
```

Comment:
I would prefer to use non-word characters (`\W`). It's weird if `name@gmail.com` will trigger suggestions (by the way good case for tests).

Reply:
Hey @TrySound I've had another look at this, not too sure what you mean? 
The regex I used wouldn't trigger with something like `name@gmail.com`.

Anyway, is this the regex you prefer? 
`new RegExp(`(\\W|^)${escapeRegExp(trigger)}${regExp}`, 'g');`